docker/README.md

```diff
@@ -0,0 +1,51 @@
+# docker files
+
+This is a collection of Docker image build files for various purposes.
+
+## Building an image
+
+Builds need to run from the top-level directory since the sources will be
+copied to the builder image during the course of the build.
+
+Build the image by referencing the respective docker file from the repository
+root:
+
+```shell
+$ docker build [-t <tag>] -f docker/<dockerfile> .
+```
+e.g.
+```shell
+$ docker build -t cache-stresser -f docker/Dockerfile.cache-stresser .
+```
+
+After the build concluded, you may push the image to the registry.
```

Comment:
This "concluded" sounds weird, it's past tense for something that's actually in the future... I think "is done" would sound better.

Reply:
The whole sentence feels out of place; I'll remove the registry reference altogether.